sagemaker-python-sdk/dgl_kge/kge_wikimedia.ipynb

```diff
@@ -0,0 +1,1396 @@
+            "-------------- Test result --------------\n",
+            "Test average MRR: 0.4159753346227368\n",
+            "Test average MR: 1001.1689418833716\n",
+            "Test average HITS@1: 0.3540242971873324\n",
+            "Test average HITS@3: 0.45541123141672746\n",
+            "Test average HITS@10: 0.5213350742247359\n",
+            "-----------------------------------------\n",
+            "Test takes 6781.685 seconds\n",
+            "CPU times: user 2min 7s, sys: 30 s, total: 2min 37s\n",
+            "Wall time: 2h 9min 59s\n"
+          ]
+        }
+      ],
+      "execution_count": 1,
+      "metadata": {}
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Let's run the inference command line. In our case we want to find the Top 5 similar nodes to our \"head.list\" file. Our file contains the following entries:\n",
+        "\n",
+        "Jeff Bezos\n",
+        "Barack Obama"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Using PyTorch Backend\n",
+        "!DGLBACKEND=pytorch dglke_score --data_path ./data/wikimedia/ --model_path ./wikimedia/TransE_l2_wikimedia_0/ \\\n",
+        "--format 'h_*_*' --data_files head.list \\\n",
+        "--score_func none --topK 5"
+      ],
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Traceback (most recent call last):\n",
+            "  File \"/usr/local/bin/dglke_score\", line 11, in <module>\n",
+            "    load_entry_point('dglke==0.1.0.dev0', 'console_scripts', 'dglke_score')()\n",
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 490, in load_entry_point\n",
+            "    return get_distribution(dist).load_entry_point(group, name)\n",
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 2854, in load_entry_point\n",
+            "    return ep.load()\n",
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 2445, in load\n",
+            "    return self.resolve()\n",
+            "  File \"/usr/lib/python3.6/dist-packages/pkg_resources/__init__.py\", line 2451, in resolve\n",
+            "    module = __import__(self.module_name, fromlist=['__name__'], level=0)\n",
+            "  File \"/usr/local/lib/python3.6/site-packages/dglke-0.1.0.dev0-py3.6.egg/dglke/infer_score.py\", line 25, in <module>\n",
+            "  File \"/usr/local/lib/python3.6/site-packages/dglke-0.1.0.dev0-py3.6.egg/dglke/models/__init__.py\", line 20, in <module>\n",
+            "  File \"/usr/local/lib/python3.6/site-packages/dglke-0.1.0.dev0-py3.6.egg/dglke/models/general_models.py\", line 30, in <module>\n",
+            "ModuleNotFoundError: No module named 'dgl'\n"
```

Comment:
Why do you think it breaks down here when it worked earlier?

Reply:
The inference / score function was added to the dgl-ke pip package was added later on. I had python package dependencies challenges. I am re-running now the whole notebook from a clean environment and I will commit a new version.

Reply:
@phiamazon Any status update? Did the notebook break after you re-ran the notebook?